### PR TITLE
Add redirect option to external auth signup component

### PR DESCRIPTION
**Pull request by pjlast**
Adds a `redirect` option to the `ExternalsAuth` component which can be used to specify a redirect page after a user signs-up successfully from specific locations.

### CODE REVIEW

File: client/web/src/get-cody/GetCodyPage.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import 'wildcard/src/global-styles/breakpoints';
 
-@font-face{
+@font-face {
```
Comment (mrnugget) [inline client/web/src/get-cody/GetCodyPage.module.scss:3]: Should this be in here?

**Review by mrnugget — APPROVED** [review]
Makes sense -- this way it only renders on dotcom, right?

File: client/web/src/get-cody/GetCodyPage.module.scss (hunk 1 above)
Comment (pjlast) [reply client/web/src/get-cody/GetCodyPage.module.scss:3]: It was done by `pnpm format`

Comment (pjlast) [reply client/web/src/get-cody/GetCodyPage.module.scss:3]: And judging by the rest of the `.scss` files, the style is to have a space there so 🤷‍♂️

**Pull request merged**